README.md

```diff
@@ -27,6 +27,7 @@ Awesome DeFi apps you can deploy on Akash
 - [Balancer](Balancer)
 - [Luaswap](luaswap)
 - [SushiSwap](SushiSwap)
+- [Yearn.finance](https://github.com/iearn-finance/iearn-finance)
```

Comment:
Please link to the directory here like the others do.  Thanks!

Reply:
OK,i have changed the link.

Reply:
it's still not quite right.  Look at sushi swap above.  it should link to `Yearn.finance` or whatever the name of the directory you created here is.

Reply:
OK,i have changed the link.